File: python/week07/Week07 Python/Square.py
from Point import Point


class Square(Point):

    def __init__(self, x, y, s):
        super().__init__(x, y)
        self.s = s

    def __str__(self):
        return f"Sides={self.s} LeftTop=" + super().__str__()

    def area(self):
        return self.s*self.s

File: Instructor Java/Week07  Java/Week07 Java/Square.java
/**
 * Square
 */
public class Square extends Point {

  private int s = 0;

  public Square(int x, int y, int s) {
    super(x, y);
    this.s = s;
  }

  /**
   * @return the s
   */
  public int getS() {
    return s;
  }

  /**
   * @param s the s to set
   */
  public void setS(int s) {
    this.s = s;
  }

  @Override
  public double area() {
    return 1.0 * s * s;
  }

  @Override
  public String toString() {
    return String.format("Side=%d LeftRight=(%d,%d)", s, getX(), getY());
  }
}
